components/org.wso2.carbon.identity.authz.valve/src/main/java/org/wso2/carbon/identity/authz/valve/AuthorizationValve.java

```diff
@@ -67,6 +68,22 @@ public void invoke(Request request, Response response) throws IOException, Servl
                 authorizationContext.setIsCrossTenantAllowed(resourceConfig.isCrossTenantAllowed());
                 authorizationContext.setAllowedTenants(resourceConfig.getCrossAccessAllowedTenants());
             }
+
+            String requestURI = request.getRequestURI();
+            if (requestURI.startsWith("/o/")) {
+                AuthorizationResult authorizationResult =
+                        authorizeInOrganizationLevel(request, response, authenticationContext, resourceConfig,
+                                authorizationContext);
+                /*
+                If the user has organization level access, authorize and proceed.
+                Else evaluate authorization through old authz model.
```

Comment:
Is this comment correct?

Reply:
rephrase the comment in [769749c](https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/184/commits/769749cc0c334b7232150455cfb08718ff812631)
Fallback to old authz model is already added as a comment is the below line.